utils/wasm-gen/src/config/module.rs

```diff
@@ -24,7 +24,10 @@
 //! and [`ConstantParams`].
 
 use arbitrary::{Arbitrary, Result, Unstructured};
-use wasm_smith::{InstructionKind::*, InstructionKinds, SwarmConfig};
+use wasm_smith::{
+    InstructionKind::{self, *},
```

Comment:
`InstructionKind` or kinds must be `pub use` as you obligate users to set config using these types.